deeplearning4j-core/pom.xml

```diff
@@ -65,6 +65,22 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.deeplearning4j</groupId>
+            <artifactId>deeplearning4j-tsne</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.deeplearning4j</groupId>
+            <artifactId>deeplearning4j-specialized-iterators</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
```

Comment:
I'd be inclined to make this default (i.e., compile) scope not test scope to make things easier for users - my thinking is that deeplearning4j-core is "all the usual stuff you might need" - things like MNIST iterator included.
It also saves users wondering what's going on when they upgrade after release - otherwise where did their MNIST iterator go, and now how do they find out about the change?